src/deepsparse/pipeline.py

```diff
@@ -177,21 +177,7 @@ def __init__(
         self.engine = self._initialize_engine()
 
     def __call__(self, *args, **kwargs) -> Union[BaseModel, Future]:
```

Comment:
I understand that this predates the `unify-run-split-join` branch and will be updated once the branch lands?

Reply:
Yes, though I believe we'll be landing this PR first, will make sure its updated either way